src/lib/plugin.js

```diff
@@ -365,18 +381,36 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   * _fulfillCondition (transferId, conditionFulfillment) {
-    const fulfillmentRes = yield this._request({
-      method: 'put',
-      uri: this.host + '/transfers/' + transferId + '/fulfillment',
-      body: conditionFulfillment,
-      json: false
-    })
+    try {
+      cc.fromFulfillmentUri(conditionFulfillment)
```

Comment:
Is it worth having a dependency on `five-bells-condition` just to throw this error? The ledger will do this check and return the error anyway